### PR TITLE
Constitutional Amendments Draft

### PR DESCRIPTION
**Pull request by bradleysigma**
Constitutional Amendments require a 3/4 vote at a general meeting to implement.

Feel free to make suggestions. Simple edits I'll make right away. Other edits will be voted on at the general meeting.

### CODE REVIEW

File: constitution.tex [hunk 1]
```diff
@@ -0,0 +1,328 @@
...
+	\begin{enumerate}
+	\item fails to comply with any of the provisions of the Constitution;
+	\item has membership fees in arrears;
+	\item violates the Society's Code of Conduct;
+	\item acts in a manner considered to be injurious or prejudicial to the character of interests of the Society;
+	\item ``yeet"s excessively;
+	\item ``\includegraphics[height=1em]{y.png}"s excessively; or
+	\item dabs excessively;
+	\item deletes the \texttt{:dab:} emoji; or
+	\item uses \textit{Comic Sans}
```
Comment (CameronAavik) [inline constitution.tex]: While I appreciate the humour, not sure if these items should be in the constitution

**Comment by TRManderson** [comment]
Is there any particular reason for a new constitution?

**Comment by ashfordneil** [comment]
Why change away from markdown?

**Comment by CameronAavik** [comment]
Personally I feel uncomfortable with a constitution that is not a diff of the existing one as it is impossible to tell what is different except by reading the entire constitution and comparing with the old one to look for the relevant section that was modified.

When I read the constitution draft that was posted a while back, I didn't see anything in particular that indicated we needed to change all the sections up either.

**Comment by jgat** [comment]
Why have a markdown file and a TeX file and the rendered PDF? Now you have to keep all three in sync. Consider also using pandoc to produce a LaTeX rendering from Markdown, and including a Makefile to build the PDF.

+1 to Cameron's comment - the key reason we put the constitution in version control is so that any member can easily read the diff to understand how each amendment changes the content of the constitution.

If you must reformat the document, please do so in a way that it's easy to check no material differences have been introduced (e.g. the rendered diff shows that only whitespace has changed). And if you wish to change the content, do so separately from the reformatting, so that voting members can easily see the proposed changes.

File: constitution.md [hunk 2]
```diff
@@ -1,299 +1,230 @@
...
+* being given a requisition in writing signed by no less than one--third (1/3) of the members presently on the Management Committee, and no less than two (2) members of the Management Committee, clearly stating the reasons why said Special General Meeting is being convened and the nature of the business to be transacted thereat;  
+* being given a requisition in writing signed by members of the Society numbering not less than the number of members required to constitute quorum at a general meeting, clearly stating the reasons why said Special General Meeting is being convened and the nature of the business to be transacted thereat; or  
+* being given a notice in writing of an intention to appeal against the decision of the Management Committee to reject an application for membership or to terminate the membership of any member.  
+  
+### 18 Quorum at General Meeting  
+18.1 At any General Meeting, the number of members required to constitute a quorum shall be a number of members equal to the smallest integer greater than or equal to the square root of one hundred and twenty--five plus the number of financial members of the Society (?v(125+n)?).
```
Comment (CameronAavik) [inline constitution.md]: The quorum requirements aren't showing correctly
>(?v(125+n)?)

**Comment by CameronAavik** [comment]
Also reposting my comment from UQCS Slack so it's visible here. Requiring LaTeX means that anyone who wants to make a PR needs to know it. Not sure if we should have that dependency.

File: constitution.md (hunk 2 above)
Comment (CameronAavik) [reply constitution.md]: Still looks broken to me after the changes

**Comment by bradleysigma** [comment]
Each section has now been split into its own pull request.

**Pull request closed without merging**